tests/unit/test_account.py

```diff
@@ -68,3 +69,201 @@ def test_manual_workspace_info(mocker):
         wir.manual_workspace_info(prompts)
 
     ws.workspace.upload.assert_called()
+
+
+def test_create_acc_groups_should_create_acc_group_if_no_group_found_in_account(mocker):
+    acc_client = create_autospec(AccountClient)
+    acc_client.config = Config(host="https://accounts.cloud.databricks.com", account_id="123", token="123")
+
+    acc_client.workspaces.list.return_value = [
+        Workspace(workspace_name="foo", workspace_id=123, workspace_status_message="Running", deployment_name="abc")
+    ]
+
+    ws = create_autospec(WorkspaceClient)
+
+    def workspace_client(**kwargs) -> WorkspaceClient:
+        return ws
+
+    account_workspaces = AccountWorkspaces(acc_client, workspace_client)
+
+    group = Group(
+        id="12",
+        display_name="de",
+        members=[ComplexValue(display="test-user-1", value="20"), ComplexValue(display="test-user-2", value="21")],
+    )
+
+    ws.groups.list.return_value = [group]
+    ws.groups.get.return_value = group
+    acc_client.groups.create.return_value = group
+
+    account_workspaces.create_account_level_groups()
+
+    acc_client.groups.create.assert_called_with(
+        display_name="de",
+    )
+    acc_client.groups.patch.assert_called_with(
+        "12",
+        operations=[
+            iam.Patch(
+                op=iam.PatchOp.ADD,
+                path='members',
+                value=[{'display': 'test-user-1', 'value': '20'}, {'display': 'test-user-2', 'value': '21'}],
```

Comment:
Isn't just value enough there?

Reply:
Test will fail, have to specify all parameters ...